tests/unit/test_charm.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Mariyan Dimitrov
+# See LICENSE file for licensing details.
+#
+# Learn more about testing at: https://juju.is/docs/sdk/testing
+
+import unittest
+
+import ops.testing
+from charm import MatrixOperatorCharm
+from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
+from ops.testing import Harness
+
+
+class TestCharm(unittest.TestCase):
```

Comment:
Should this be written in pytest rather than unittest?

Reply:
It should, `charmcraft` still generates tests in `unittest`, I haven't changed it yet.